internal/btf/btf_types.go

```diff
@@ -61,6 +62,45 @@ type btfType struct {
 	SizeType uint32
 }
 
+func (k btfKind) String() string {
+	switch k {
+	case kindUnknown:
+		return "Unknown"
+	case kindInt:
+		return "Int"
+	case kindPointer:
+		return "Pointer"
+	case kindArray:
+		return "Array"
+	case kindStruct:
+		return "Struct"
+	case kindUnion:
+		return "Union"
+	case kindEnum:
+		return "Enum"
```

Comment:
I'd use `Enumeration` here in order to stay consistent with the comment in `linux/btf.h`.
```suggestion
		return "Enumeration"
```